src/views/Login.jsx

```diff
@@ -1,35 +1,95 @@
-import React from 'react';
-import Container from 'react-bootstrap/Container';
-import Form from 'react-bootstrap/Form';
-import Button from 'react-bootstrap/Button';
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import Container from "react-bootstrap/Container";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
 
 /**
  * Implement the login functionality.
- * 
+ *
  * Given a username and password, check the database for a match.
  * If a match is found, log the user in.
- * 
+ *
  * Use the following endpoint to check for a username and password match:
  * POST https://api-demo.kingslandtesting.com/user/login
- * 
+ *
  * Then, redirect appropriately to /dashboard and adjust the sidebar menuitems accordingly.
  */
 const Login = () => {
+  const navigate = useNavigate();
+  const [data, setData] = useState({
+    username: "",
+    password: "",
+  });
+
+  const handleChange = (e) => {
+    setData({ ...data, [e.target.name]: e.target.value });
+  };
+
+  const navigateTo = (path) => {
+    setTimeout(() => {
+      navigate(path, { replace: true });
+      window.location.reload(false);
+    }, 500);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    const userData = {
+      username: data.username,
+      password: data.password,
+    };
+
+    await axios
+      .post("https://api-demo.kingslandtesting.com/user/login", userData, {
+        "Content-Type": "application/json",
+      })
+      .then((response) => {
+        console.log(response.data);
+        const loggedInUserName =
+          response.data.firstName + " " + response.data.lastName;
+        if (response.data) {
+          localStorage.setItem("Authentication", "true");
+          localStorage.setItem("Name", loggedInUserName);
+          navigateTo("/dashboard");
+        }
+      })
+      .catch((error) => {
+        console.log(error);
+        localStorage.setItem("Authentication", "false");
+        navigateTo("/login");
```

Comment:
Error isn't handled properly. No prompts to the user that the credentials are invalid.